span-normalizer/span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanPreProcessorTest.java

```diff
@@ -27,7 +30,7 @@ void testPreProcessSpan_missingTenantId() {
           // span dropped since tenant detail not present
           String tenantId = "tenant-" + random.nextLong();
           Map<String, Object> configs = new HashMap<>(getCommonConfig());
-          configs.putAll(Map.of("processor", Map.of()));
+          configs.putAll(Map.of("processor", Map.of("late.arrival.threshold.duration", "1d")));
```

Comment:
As the config - `late.arrival.threshold.duration` - is part of the mandatory config, need to update all existing tests.